### PR TITLE
Upgrade to version 2.7.0 of OpenSearch Dashboards

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
Upgrade to version 2.7.0 of OpenSearch Dashboards, fixes test added after tag 2.4.1.
 
### Issues Resolved
- #62 
 
### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
  - [x] `yarn test:jest_integration`
  - [x] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,9 @@
     "number": 8467,
     "sha": "6cb7fec4e154faa0a4a3fee4b33dfef91b9870d9"
   },
+  "wazuh": {
+    "version": "4.4.0"
```
Comment (AlexRuiz7) [inline package.json:23]: Bump to 4.5.0

Tests and snapshots should use `4.x.x`, `x.y.z` or similar.

Comment (yenienserrano) [reply package.json]: Done

**Review by Desvelao — APPROVED** [review]
**review**

code :green_circle:

File: src/core/public/chrome/ui/header/header_help_menu.tsx [hunk 2]
```diff
@@ -208,7 +208,7 @@ class HeaderHelpMenuUI extends Component<Props, State> {
         <EuiButtonEmpty href={opensearchDashboardsDocLink} target="_blank" size="xs" flush="left">
           <FormattedMessage
             id="core.ui.chrome.headerGlobalNav.helpMenuOpenSearchDashboardsDocumentationTitle"
-            defaultMessage="Documentation"
+            defaultMessage="Wazuh documentation"
```
Comment (AlexRuiz7) [inline src/core/public/chrome/ui/header/header_help_menu.tsx:211]:
```suggestion
            defaultMessage="Documentation"
```

File: src/core/public/chrome/ui/header/header_help_menu.tsx [hunk 3]
```diff
@@ -217,7 +217,7 @@ class HeaderHelpMenuUI extends Component<Props, State> {
         <EuiButtonEmpty href={helpSupportUrl} target="_blank" size="xs" flush="left">
           <FormattedMessage
             id="core.ui.chrome.headerGlobalNav.helpMenuAskOpenSearchTitle"
-            defaultMessage="Community"
+            defaultMessage="Ask Wazuh"
```
Comment (AlexRuiz7) [inline src/core/public/chrome/ui/header/header_help_menu.tsx:220]:
```suggestion
            defaultMessage="Community"
```

**Review by AlexRuiz7 — APPROVED** [review]

**Comment by AlexRuiz7** [comment]
Merging blocked as we need to integrate #63 first

**Review by asteriscos — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -19,6 +19,9 @@
     "number": 8467,
     "sha": "6cb7fec4e154faa0a4a3fee4b33dfef91b9870d9"
   },
+  "wazuh": {
+    "version": "4.7.0"
```
[the remark this answers is not in the document]
Comment (yenienserrano) [reply package.json:23]: From what I understand, the fork was passed for the old 4.6 (current 4.7).

[the remark this answers is not in the document]
Comment (AlexRuiz7) [reply package.json:23]: It was from 4.5.0 to 4.6.0. Correct value is 4.6.0

**Pull request merged**